output_delegation.go

```diff
@@ -143,8 +138,7 @@ func (d *DelegationOutput) VBytes(rentStruct *RentStructure, _ VBytesFunc) VByte
 		rentStruct.VBFactorData.Multiply(serializer.SmallTypeDenotationByteSize+serializer.UInt64ByteSize*4) +
```

Comment:
Mhh your current commit doesn't contain the workscore changes. You should pull in current `serix-iota-core` and remove the immutable features there as well.